services/bettersocial/fcmToken/sendMultiDeviceCommentNotification.js

```diff
@@ -24,10 +24,19 @@ const sendMultiDeviceCommentNotification = async (
         data: {
           feed_id: activity_id,
           type: 'feed'
-        }
+        },
+        token: user?.token
       };
 
-      await messaging().sendToDevice(user?.token, payload);
+      try {
+        await messaging().send(payload);
+      } catch (error) {
+        if (error.code === 'messaging/registration-token-not-registered') {
+          console.error(`Token ${user?.token} removed from database: ${error.message}`);
+        } else {
+          console.error(`Failed to send message: ${error.message}`);
+        }
+      }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The error handling here is good as it catches specific errors and logs relevant messages. However, it only logs the error and does not handle it. Depending on the application's requirements, you might want to consider rethrowing the error after logging it, or handling it in some other way.
```diff
  try {
    await messaging().send(payload);
  } catch (error) {
    if (error.code === 'messaging/registration-token-not-registered') {
      console.error(`Token ${user?.token} removed from database: ${error.message}`);
+     throw error;
    } else {
      console.error(`Failed to send message: ${error.message}`);
+     throw error;
    }
  }
```
Please note that this suggestion may not be applicable if the intention is to continue execution even when an error occurs.


<!-- This is an auto-generated comment by OSS CodeRabbit -->